### PR TITLE
2021 AGM agenda

### PR DESCRIPTION
**Pull request by GregBrimble**
This adds the proposed voting mechanisms to be voted for in the 2021 AGM.

### CODE REVIEW

File: 2021-agm.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Homebrew Annaul General Meeting 2021
```
Comment (sjackman) [inline 2021-agm.md:1]:
```suggestion
# Homebrew Annual General Meeting 2021
```

File: 2021-agm.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Voting Mechanism
+
+### Project Leader
+
+The Homebrew Project Leader will be chosen by holding an [Schulze Condorcet](https://en.wikipedia.org/wiki/Schulze_method) election. This popular method of voting is used by several organizations such as Wikimedia, Debian and Ubuntu. The single highest-ranked candidate will be elected to become the Project Leader.
```
Comment (nandahkrishna) [inline 2021-agm.md:7]:
```suggestion
The Homebrew Project Leader will be chosen by holding a [Schulze Condorcet](https://en.wikipedia.org/wiki/Schulze_method) election. This popular method of voting is used by several organizations such as Wikimedia, Debian and Ubuntu. The single highest-ranked candidate will be elected to become the Project Leader.
```

File: 2021-agm.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Homebrew Annaul General Meeting 2021
+
+## Voting Mechanism
```
Comment (sjackman) [inline 2021-agm.md:3]:
```suggestion
# Motions
## Motion to adopt the voting system
```

File: 2021-agm.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Homebrew Annaul General Meeting 2021
+
```
Comment (sjackman) [inline 2021-agm.md:2]:
```suggestion

# Agenda

- Call to order
- Adoption of the agenda
- Project Leader's report
- Project Leadership Committee's report
- Technical Steering Committee's report
- Treasurer's report of the financial statements
- Motion to adopt the voting system
- Election of the Project Leader
- Election of the Project Leadership Committee
- Member presentations
- Adjournment
```

File: 2021-agm.md (hunk 2 above)
Comment (sjackman) [inline 2021-agm.md:7]:
```suggestion
The Homebrew Project Leader will be chosen by holding an [Schulze Condorcet method](https://en.wikipedia.org/wiki/Schulze_method) election. This popular method of voting is used by several organizations such as Wikimedia, Debian, and Ubuntu. The single highest-ranked candidate, who is preferred over every other candidate in pairwise comparisons, will be elected to become the Project Leader.
```

File: 2021-agm.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+### Project Leader
+
+The Homebrew Project Leader will be chosen by holding an [Schulze Condorcet](https://en.wikipedia.org/wiki/Schulze_method) election. This popular method of voting is used by several organizations such as Wikimedia, Debian and Ubuntu. The single highest-ranked candidate will be elected to become the Project Leader.
+
+Voting by proxy is permitted, and proxy votes count towards the quorum for the election.
+
+### Project Leadership Committee (PLC)
+
+The Homebrew Project Leadership Committee will be chosen by holding a [Meek Single Transferable Vote (STV)](https://en.wikipedia.org/wiki/Counting_single_transferable_votes#Meek) election. The quota (threshold) of votes for a candidate to be elected will be calculated using the [Droop quota formula](https://en.wikipedia.org/wiki/Droop_quota).
```
Comment (sjackman) [inline 2021-agm.md:13]:
```suggestion
The Homebrew Project Leadership Committee will be chosen by holding a [Meek Single Transferable Vote (STV)](https://en.wikipedia.org/wiki/Counting_single_transferable_votes#Meek) election. The quota (threshold) of votes for a candidate to be elected will be calculated using the [Droop quota](https://en.wikipedia.org/wiki/Droop_quota).
```

**Review by sjackman — CHANGES_REQUESTED** [review]
Thanks for starting this document!

File: 2021-agm.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- Election of the Project Leader
+- Election of the Project Leadership Committee
+- Project Leader's report
+- Project Leadership Committee's report
+- Technical Steering Committee's report
+- Treasurer's report of the financial statements
+- Member presentations
+- Adjournment
+# Motions
+## Motion to adopt the voting system
```
Comment (sjackman) [inline 2021-agm.md:17]:
```suggestion
- Adjournment

# Motions

## Motion to adopt the voting system
```

**Review by sjackman — APPROVED** [review]
Thank you, Greg!

File: 2021-agm.md (hunk 6 above)
Comment (GregBrimble) [reply 2021-agm.md]: Sorry, forgot to `git add` before making doing those changes. Thanks for catching.

File: 2021-agm.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+- Call to order
+- Adoption of the agenda
+- Motion to adopt the voting system
+- Election of the Project Leader
+- Election of the Project Leadership Committee
+- Project Leader's report
+- Project Leadership Committee's report
+- Technical Steering Committee's report
+- Treasurer's report of the financial statements
```
Comment (MikeMcQuaid) [inline 2021-agm.md:13]: Can we put these before the elections? Two reasons:
- the PL/PLC/TSC may want to state things that affect the elections
- the PL/PLC/TSC may not be reelected and might not want to give the same presentation if that's the case 😂

**Review by SMillerDev — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Review by sjackman — APPROVED** [review]

**Pull request merged**